src/custom/state/claim/hooks/utils.ts

```diff
@@ -161,17 +164,18 @@ export type PaidClaimTypeToPriceMap = {
 
 /**
  * Helper function to get vCow price based on claim type and chainId
- *
- * @param type
  */
 export function claimTypeToTokenAmount(type: ClaimType, chainId: SupportedChainId) {
   switch (type) {
     case ClaimType.GnoOption:
       return { token: GNO[chainId], amount: GNO_PRICE }
-    case ClaimType.Investor:
-      return { token: USDC, amount: USDC_PRICE }
+    case ClaimType.Investor: {
```

Comment:
this will probably change with my USDC approve pr #2154 as i created a USDC map, tho maybe not idk

Reply:
Yep, that is another way of fixing it

Reply:
Would you rather I remove it then?

Reply:
Yours is definitely more complete as I missed the USDC_XDAI. 

Should use that instead. Can you apply the fix in your PR? I'll remove this part from mine

Reply:
Ok ok, as agreed I'll adjust after https://github.com/gnosis/cowswap/pull/2154 is merged